client/src/holdings/HoldingsList.tsx

```diff
@@ -62,9 +62,10 @@ export default function HoldingsList({user}: HoldingsListProps) {
     setOpen([...open]);
   };
 
-  const addHolding = (holding: Holding) => {
-    holdings.push(holding);
-    setHoldings([...holdings]);
+  const addHolding = (holding: Holding) => { 
+    holdings.push(holding); 
+    holdings.sort((a, b) => (a.name > b.name) ? 1 : -1) 
```

Comment:
Nice one! Only issue is that this does not ignore case, so it results in a list with all the lowercase-starting holdings at the bottom. So just fix it so it ignores case in sorting and then it should be good.

Also add a semicolon at the end of the line for style consistency.